controllers/userController.js

```diff
@@ -0,0 +1,151 @@
+const express = require("express");
+const User = require("../model/UserModel");
+const asyncHandler = require("express-async-handler");
+const bcyrpt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+// creating a register endpoint
+
+const registerUser = asyncHandler(async (req, res) => {
+  const { username, email, password } = req.body;
+  if (!username || !email || !password)
```

Comment:
Update the user table
To create an account, we need firstname, lastname, username, email, phone and password